pkg/kubecost/allocation.go

```diff
@@ -119,6 +119,7 @@ type LbAllocation struct {
 	Service string  `json:"service"`
 	Cost    float64 `json:"cost"`
 	Private bool    `json:"private"`
+	Ip      string  `json:"ip"`
```

Comment:
didn't he already make the bingen changes? he would just have to run the bingen tag number on this specific field to ignore it?

Reply:
Yeah, I definitely don't understand when to ignore vs. when to add to bingen and bump up the version

Reply:
You only ignore if you do not want the field to be retained in filestorage. I don't think that is the case here.
Whenever you add a new field to a bingen struct you need add an ignore tag or increment the group version and add a version tag, then run bingen again.

If you look at the codec changes on this pr this field is read off the buffer without a version check, doing this will cause errors when unmarshalling binary that does not have the field.

Reply:
ah right. @nickcurie here is an example of us setting a bingen version on a field https://github.com/opencost/opencost/blob/develop/pkg/kubecost/allocation.go#L74